docs/plugins/repository-s3.asciidoc

```diff
@@ -251,8 +251,12 @@ PUT _snapshot/my_s3_repository
 The following settings are supported:
 
 `bucket`::
-
-    The name of the bucket to be used for snapshots. (Mandatory)
+(Required)
+Name of the S3 bucket to use for snapshots.
++
+The bucket name must adhere to the
+https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html[Amazon
```

Comment:
I think the correct link for the naming requirements is https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html#bucketnamingrules

Reply:
Good call. Updated with c3c299dde9fe96e283266353661b9eb7cf36257d.